### DISCUSSION TITLE
ProductGroup.HMIPW  in create_central_links hinzufügen

### DISCUSSION THREAD

**Discussion by Revilo91** [SukramJ/aiohomematic#1880] [General]
In der Methode `relevant_for_central_link_management` sollte noch die `ProductGroup.HMIPW` hinzugefügt werden. 

Es wird seit einem der letzten Updates (Raspberrymatic) keine Tasterdrücke mehr bei einem HmIPW-DRI32 weitergegeben. Dies müsste händisch in der Raspberrymatic eingestellt werden.

**Comment by SukramJ** [comment]
Danke. Hättest ja auch gleich einen PR erstellen können (Mach ich jetzt).

**Comment by Revilo91** [comment]
Oh, ich dachte man schreibt erst in eine Diskussion 😅

Danke dir 👍
